testnets/galacticatestnet/chain.json

```diff
@@ -0,0 +1,125 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "galacticatestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "Galactica Protocol Testnet",
+  "chain_id": "galactica_9301-1",
+  "bech32_prefix": "gala",
+  "daemon_name": "galacticad",
+  "node_home": "$HOME/.galactica",
+  "key_algos": [
+    "ethsecp256k1"
+  ],
+  "slip44": 60,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "agnet",
+        "fixed_min_gas_price": 10,
```

Comment:
Doesn't make sense to have a chain minimum above the recommended rates described below.